iscsi/osfamilymap.yaml

```diff
@@ -39,9 +39,9 @@ Debian:
     pkgs:
       wanted:
         - linux-image-extra-virtual
```

Comment:
Huh, `linux-image-extra-virtual` isn't either a Debian package. Now that this one shows up, they are *Ubuntu*-specific. Mind you remove this one too, so we merge this PR?

Reply:
Thanks for clarifying that.  done!